nhais-adaptor/edifact/incoming/parser/creators.py

```diff
@@ -0,0 +1,71 @@
+from edifact.incoming.models.interchange import InterchangeHeader

```

Comment:
Given that there is a top-level `adaptor.outgoing` package for the adaptors that create the outgoing messages, would it make sense for this `parser` package to live at the top level too?

Reply:
Do you mean to move this `parser` module to -> `adaptor/incoming`?

Reply:
Yes, sorry, that's a much clearer way of putting it!

Reply:
As discussed offline, the `adaptor` module is focused on FHIR conversion, so it wouldn't make sense to move this EDIFACT parser there. I'm happy to leave it where it is.